components/senza.rst

```diff
@@ -477,6 +477,24 @@ This component supports the following configuration properties:
 ``VolumeSize``
     How Much GB should this EBS have?
 
+**SpotPrice**
```

Comment:
can you also add the property below `InstanceType` in the definition list above?